wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -293,6 +296,18 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
+type ResetFailedPtxs =
+       "txs"
+    :> "resubmission"
+    :> "reset"
+    :> WRes Get ()
+
+type CancelApplyingPtxs =
+       "txs"
+    :> "resubmission"
+    :> "cancel"
+    :> WRes Get ()
```

Comment:
Returning `()` will screw up our swagger specs (we should return `NoContent` instead). That's probably OK here if there endpoints are short-lived anyway 😉

Reply:
Sorry, it's difficult to do now because it's first appearance of `NoContent` in this branch. I'm scared with possible conflicts of inserted `NoContents` everywhere if we're going to cherry-pick this.
It seems easier to resolve the problem with specs once it arises :slightly_smiling_face:

Reply:
Don't worry, for now specs are the least of our priorities 😉